script/generate_build_info.py

```diff
@@ -134,16 +134,15 @@ def get_maintainer_from_dockerfile(dir_name):
 def get_default_build_script(build_scripts_versions):
     result=[]
     for data in build_scripts_versions:
-        if 'ubi_8.7' in data['file']:
+        if 'ubi_9.3' in data['file']:
             return [data['version'],data['file']]
         else:
             result.append((data['version'],data['file']))
     return max(result,key=lambda x:x[0]) 
 
-
-maintainer=get_maintainer_for_package(dir_name)
-if maintainer=="Unknown":
-    maintainer= get_maintainer_from_dockerfile(dir_name)
+maintainer=input("Enter GitHub username:")
```

Comment:
hey.. we should be explicit to share github user name for `github.com`.. can be confusing for someone with GHE

Reply:
also do we need validate to validate id's existence before accepting it if possible with some api

Reply:
Will check if the validation of GitHub id is possible using Api.